nav2_behavior_tree/plugins/condition/goal_reached_condition.cpp

```diff
@@ -57,10 +59,6 @@ void GoalReachedCondition::initialize()
 {
   node_ = config().blackboard->get<rclcpp::Node::SharedPtr>("node");
 
-  nav2_util::declare_parameter_if_not_declared(
-    node_, "goal_reached_tol",
-    rclcpp::ParameterValue(0.25));
-  node_->get_parameter_or<double>("goal_reached_tol", goal_reached_tol_, 0.25);
```

Comment:
if we remove this, then our branch diverges (functionality wise) from mainline.
can we make it so it will take the input port preferentially if set?  Then you could possibly get this merged upstream while preserving existing functionality